src/devices/Adg731/Adg731.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Device.Spi;
+
+namespace Iot.Device.Adg731
```

Comment:
namespace should be `Iot.Device.Adc` - this will make it easier to use since class name and namespace have different name

Reply:
Actually reading the code further and docs a bit this should be int `.Multiplexer` namespace. Also device should expose way of selecting input channel with API which should be crystal clear i.e.:
`int SelectedChannel { get; set; }`